iMichaelOwolabi/my-diarium-backend
# 167689354 Setup test for project

#### What does this PR do?
Setup testing for project
#### Description of Task to be completed?
* Configure test for project
#### How should this be manually tested?
Clone the project repo and cd into the directory
- run npm  install to install all dependencies
- run npm test
- You should see the test file run and pass
#### Any background context you want to provide?
This task is just to setup basic testing for the project
#### What are the relevant pivotal tracker stories?
[#167689354](https://www.pivotaltracker.com/story/show/167689354)
